elyra/metadata/schemas/airflow.json

```diff
@@ -40,6 +40,14 @@
             "category": "Airflow Pipelines"
           }
         },
+        "github_api_endpoint": {
+          "title": "Github API Endpoint",
+          "description": "The Github Server URL / API endpoint - Public or Enterprise",
+          "type": "string",
```

Comment:
Does this need a `"format": "uri",` entry since it's an endpoint?

Reply:
ack, thought I had that, done